config.yml

```diff
@@ -77,6 +77,8 @@ genesis:
             requiredAllocations: "10"
             benefits:
               maxBidAmount: "30000"
+        registrationPeriod: "604800"
+        withdrawalDelay: "1209600"
```

Comment:
We should add comments to precise what it represent
Is `registrationPeriod == 1/3 unbondingPeriod` and `withdrawalDelay == 2/3 unbondingPeriod` ?

Reply:
well, yes, considering the unnbonding period is 21 days. I'll add comments